src/react-compat.js

```diff
@@ -134,6 +134,8 @@ if (REACT013) {
   };
 }
 
+const batchedUpdates = require('react/lib/ReactUpdates').batchedUpdates;
```

Comment:
This might become an issue due to https://github.com/facebook/react/issues/6460. depending on which way they go. Hopefully this might help sway them away.